tests/writer.rs

```diff
@@ -256,23 +258,10 @@ mod tests {
         assert_eq!(new_data, data);
     }
 
-    #[test]
-    fn write_descrip_panic() {
```

Comment:
Is this not tested anymore?

Reply:
It is no longer possible because `descrip` was turned back into an 80-byte array. It was kind of what I originally had in mind, but large arrays were not as nice to work with in Rust. That changed in recent versions, so unless we witness a significant performance regression, we can save the description in a `[u8; 80]` here instead of `Vec<u8>`.

Reply:
smallvec? https://docs.rs/smallvec/latest/smallvec/